uv.lock

```diff
@@ -0,0 +1,630 @@
+version = 1
```

Comment:
Should this file be under CM? Isn't it auto-generated from `pyproject.toml` by `uv`?